tests/test_connection.py

```diff
@@ -468,24 +468,6 @@ def test_read_response_returns_cached_reply(self, mock_cache, mock_connection):
                 status=CacheEntryStatus.IN_PROGRESS,
                 connection_ref=mock_connection,
             ),
-            CacheEntry(
-                cache_key=CacheKey(command="GET", redis_keys=("foo",)),
-                cache_value=b"bar",
-                status=CacheEntryStatus.VALID,
-                connection_ref=mock_connection,
```

Comment:
Why do you need to remove those entries?

Reply:
In the code below, `mock_cache.get` is only called three times, so I removed it. I hope my understanding is correct.

Reply:
`mock_cache.get` called multiple times because of cache logic, there's a "soft" lock to avoid race condition so when you're sending a command a temporary cache entry is created with status "IN_PROGRESS", then when the actual response is received the same instance becomes associated with value and status changes to "VALID". So that's why status of first instance is different from consequent calls. Also additional calls is just to verify that value is retrieved from cache

Reply:
Thank you for your patience in answering my questions, now I fully understand！

Reply:
Could you please revert those changes? And we're good to go

Reply:
done